code/modules/mob/living/simple_animal/spawner.dm

```diff
@@ -8,7 +8,8 @@
 	var/max_mobs = 5
 	var/spawn_delay = 0
 	var/spawn_time = 300 //30 seconds default
-	var/mob_type = /mob/living/simple_animal/hostile/carp
+	var/mob_types = list(/mob/living/simple_animal/hostile/carp)
+	var/chosen_mob_type
```

Comment:
Why is this on the object and not localized within the proc?

Reply:
yeah i can move it over sure